chat-commands.js

```diff
@@ -2551,6 +2551,7 @@ exports.commands = {
 				Chat.uncacheTree('./tournaments');
 				global.Tournaments = require('./tournaments');
 				Tournaments.tournaments = runningTournaments;
+				Object.assign(Chat.commands, require('./chat-plugins/github').commands);
```

Comment:
I still don't understand why stuff like this is necessary.

We should already automatically install chat plugins......